esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/opt/rose-suite-metoffice.conf

```
@@ -2,6 +2,8 @@
 DRS_CMIP6="BADC"
 DRS_OBS4MIPS="default"
 MODULE_NAME="scitools/community/esmvaltool/2.6.0"
-ROOTPATH_CMIP6="/project/champ/data/CMIP6"
+ROOTPATH_CMIP6="/project/cma/esmvaltool/CMIP6"
+ROOTPATH_CMIP5="/project/champ/data/cmip5/output1"
+ROOTPATH_OBS="/project/cma/esmvaltool/obs"
```

This path should be the same as the `OBS4MIPS` path below:
```suggestion
ROOTPATH_OBS="data/users/esmval/ESMValTool/temporary/obs/"
```